internal/pkg/node/datastructure.go

```diff
@@ -8,65 +18,61 @@ Structure of which goes to disk
 */
 type NodeYaml struct {
 	WWInternal   int `yaml:"WW_INTERNAL,omitempty" json:"WW_INTERNAL,omitempty"`
-	NodeProfiles map[string]*NodeConf
+	NodeProfiles map[string]*ProfileConf
 	Nodes        map[string]*NodeConf
+	nodeProfiles map[string]*ProfileConf
+	nodes        map[string]*NodeConf
```

Comment:
This needs to be consolidated or otherwise updated to not have both exported and non-exported names the same. (I think we can just do everything with the exported names and remove the non-exported, though callers will need to be updated.)